doc/dev/delayed-delete.rst

```diff
@@ -2,11 +2,12 @@
  CephFS delayed deletion
 =========================
 
-When you delete a file, the data is not immediately removed. Each
-object in the file needs to be removed independently, and sending
-``size_of_file / stripe_size * replication_count`` messages would slow
-the client down too much, and use a too much of the clients
-bandwidth. Additionally, snapshots may mean some objects should not be
-deleted.
+The deletion of a file does not immediately remove its data. Each
+object in the file must be removed independently. Sending
+``size_of_file / stripe_size * replication_count`` messages slows
+the client unacceptably, and consumes too much client bandwidth. 
+If snapshots exist, their existence can prevent the deletion of 
+objects associated with them.
 
-Instead, the file is marked as deleted on the MDS, and deleted lazily.
+In these cases, such files are (1) marked as deleted on the MDS and 
+(2) are deleted lazily.
```

Comment:
With the first `are` preceding `(1)`, the second following `(2)` seems non-congruent.  I might suggest "(2) deleted lazily", that is, remove the second `are`.

Reply:
I see your point.